app/src/main/java/org/simple/clinic/enterotp/EnterOtpScreen.kt

```diff
@@ -124,6 +130,12 @@ class EnterOtpScreen(
   override fun onRestoreInstanceState(state: Parcelable?) {
     super.onRestoreInstanceState(delegate.onRestoreInstanceState(state))
   }
+  
```

Comment:
This is a breaking commit. Are we purposefully doing this? 
Although, I think we can add all the overridden mobius methods in this commit.

Reply:
We should probably squash this with the next commit as we are already adding `defaultModel` here

Reply:
So with fragment migration, mostly all commits would be breaking if we see it individually. So we're anyway squashing and merging so it doesn't matter, 
I've made these as two different commits because the bindview and model overridden methods were mandatory implemented from BaseScreen and that particular commit extends baseScreen and the rest of the Mobius methods into different one.